lib/apartment/connection_pool.rb

```diff
@@ -12,6 +16,7 @@ def use(spec)
       else
         establish_connection(klass, spec)
       end
+      klass.connection.enable_query_cache!
```

Comment:
By the way, I think this just returns `true`, so this may unintentionally change the return value of `#use`.